tools/biosnoop.bt

```diff
@@ -12,6 +12,7 @@
 
 #ifndef BPFTRACE_HAVE_BTF
 #include <linux/blkdev.h>
+#include <linux/blk-mq.h>
```

Comment:
This is a potential problem for systems which don't have `linux/blk-mq.h` but it's been added in kernel 3.13 which is over 8 years old, so I guess that we're fine to add it.